views/js/pciCreator/dev/mathEntryInteraction/runtime/mathEntryInteraction.js

```diff
@@ -518,13 +526,13 @@ define([
         createToolbar: function createToolbar() {
             var self = this,
                 availableTools = {
-                    frac:   { label: '<sup>x</sup>&frasl;<sub>y</sub>',latex: '\\frac',fn: 'cmd',desc: 'Fraction' },
+                    frac:   { label: 'x/y',         latex: '\\frac',    fn: 'cmd',      desc: 'Fraction' },
```

Comment:
This changes the visuals of the fraction tool, is this a requested change?

Reply:
This how it looks with a previous version: 
<img width="56" alt="Screenshot 2020-05-12 at 15 53 03" src="https://user-images.githubusercontent.com/6339002/81700330-d0465000-9468-11ea-973c-6dca499c111c.png">